GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift

```diff
@@ -20,7 +20,8 @@ final class GBHSelectedView: UIView {
     // MARK: - Private var
     private var didSetupConstraints = false
 
-    private var checkMarkView: UIImageView?
+    private var checkMarkView: UIView?
+    private let checkMarkViewSize = CGSize(width: 20, height: 20)
```

Comment:
Maybe apply ratio size is preferable ? 
Like : 
`let size = self.frame.size.width / 5
CGSize(width:  size, height: size)` ? :)

Reply:
The frame is .zero at this moment.

Reply:
Yes, that's why you should initialize it in the init :) 
I will do it.

Reply:
Alright - Thanks! 👍